nouns.json

```diff
@@ -1979,5 +2141,375 @@
+        "EU": { "atom": "", "pstn": "" },
+        "FA": { "atom": "", "pstn": "" },
+        "FI": { "atom": "", "pstn": "" },
+        "FR": { "atom": "", "pstn": "" },
+        "IT": { "atom": "", "pstn": "" },
+        "JA": { "atom": "", "pstn": "" },
+        "KO": { "atom": "", "pstn": "" },
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      },
+      "085b027c": {
+        "CA": { "atom": "de Taylor", "pstn": "after" },
+        "DE": { "atom": "", "pstn": "" },
+        "EN": { "atom": "Taylor", "pstn": "before" },
+        "ES": { "atom": "de Taylor", "pstn": "after" },
+        "EU": { "atom": "", "pstn": "" },
+        "FA": { "atom": "", "pstn": "" },
+        "FI": { "atom": "", "pstn": "" },
+        "FR": { "atom": "", "pstn": "" },
+        "IT": { "atom": "", "pstn": "" },
+        "JA": { "atom": "", "pstn": "" },
+        "KO": { "atom": "", "pstn": "" },
+        "PL": { "atom": "", "pstn": "" },
+        "PT": { "atom": "", "pstn": "" },
+        "RU": { "atom": "", "pstn": "" },
+        "TR": { "atom": "", "pstn": "" },
+        "ZH": { "atom": "", "pstn": "" }
+      }
+    }
+  },
+  "086d026b": {
+    "root": {
+      "CA": { "atom": "acnode", "gend": "masculí" },
+      "DE": { "atom": "", "gend": "" },
+      "EN": { "atom": "acnode", "gend": "" },
+      "ES": { "atom": "acnodo", "gend": "masculino" },
+      "EU": { "atom": "", "gend": "" },
+      "FA": { "atom": "", "gend": "" },
+      "FI": { "atom": "", "gend": "" },
+      "FR": { "atom": "", "gend": "" },
+      "IT": { "atom": "", "gend": "" },
+      "JA": { "atom": "", "gend": "" },
+      "KO": { "atom": "", "gend": "" },
+      "PL": { "atom": "", "gend": "" },
+      "PT": { "atom": "", "gend": "" },
+      "RU": { "atom": "", "gend": "" },
+      "TR": { "atom": "", "gend": "" },
+      "ZH": { "atom": "", "gend": "" }
+    },
+    "adjs": {}
+  },
+  "0e88034d": {
+    "root": {
+      "CA": { "atom": "addició", "gend": "femení" },
+      "DE": { "atom": "", "gend": "" },
+      "EN": { "atom": "addition", "gend": "" },
+      "ES": { "atom": "adición", "gend": "femenino" },
+      "EU": { "atom": "", "gend": "" },
+      "FA": { "atom": "", "gend": "" },
+      "FI": { "atom": "", "gend": "" },
+      "FR": { "atom": "", "gend": "" },
+      "IT": { "atom": "", "gend": "" },
+      "JA": { "atom": "", "gend": "" },
+      "KO": { "atom": "", "gend": "" },
+      "PL": { "atom": "", "gend": "" },
+      "PT": { "atom": "", "gend": "" },
+      "RU": { "atom": "", "gend": "" },
+      "TR": { "atom": "", "gend": "" },
+      "ZH": { "atom": "", "gend": "" }
+    },
+    "adjs": {}
+  },
+  "120b03a0": {
+    "root": {
+      "CA": { "atom": "adherència", "gend": "femení" },
+      "DE": { "atom": "", "gend": "" },
+      "EN": { "atom": "adherence", "gend": "" },
```

Comment:
is this "closure", as in "of a topological subset"? if so, I've never heard "adherent" used before, so maybe switch to "closure"? (and thus change the id to `0be302fe`)

Reply:
As far as I know, they are synonyms, and I'm much more used to adherence. I've read somewhere that you wanted to add this feature, so synonyms can point to each other. So, if you agree, I add closure as well and we can keep them both.

Reply:
I need to think a bit more about how to deal with synonyms, it's true...